xdsl/interactive/app.py

```diff
@@ -157,7 +158,15 @@ class InputApp(App[None]):
     text areas.
     """
 
-    def __init__(self):
+    pre_loaded_file_path_option: str | None = None
+    """
+    Saves the path name of the file called on terminal to be pre-loaded into the gui.
+    """
+
+    def __init__(
+        self,
+        pre_load_file_path: str | None,
```

Comment:
```suggestion
        input_text: str | None = "",
```
I would recommend having the file reading logic outside of the app for now. If there's an error, might as well not launch the app at all